azure_functions_worker/http_v2.py

```diff
@@ -246,14 +246,11 @@ async def catch_all(request: request_type):  # type: ignore
             from e
 
 
-async def sync_http_request(http_request, invoc_request):
+async def sync_http_request(http_request, func_http_request):
```

Comment:
Should there also be a unit test added for testing this function change or is it already covered? (eg. in `unittests/test_http_v2.py`)

Reply:
its covered in that file yes as part of e2e test